lib/stdlib/src/proc_lib.erl

```diff
@@ -522,6 +524,30 @@ translate_initial_call(DictOrPid) ->
 	    {?MODULE,init_p,5}
     end.
 
+%% -----------------------------------------------------
+%% [get] set_process_description/1
+%% Add and fetch process id's to aid in debugging
+%% -----------------------------------------------------
+
+-spec set_process_description(Id :: term()) -> ok.
+set_process_description(Id) ->
+    put('$process_description', Id),
+    ok.
+
+-spec get_process_description(Pid :: pid()) -> undefined | term().
+get_process_description(Pid) ->
+    case Pid == self() of
+        true ->
+            get('$process_description');
+        false ->
+            try get_process_info(Pid, {dictionary, '$process_description'}) of
```

Comment:
this module has 3 naming conventions for same thing: process_info, proc_info, procinfo, pinfo with various casing :-/
looks messy to me...

Note: this module seem to favour shorthands like:
- `dict` for dictionary
- `proc` for process
- `info` for information
- `proc_lib` for process_library

Maybe, if needed at all, `proc_` should be used to follow local convention.

Reply:
I have only reused old code here, I don't want to refactor the whole module for a name change.

Reply:
> I have only reused old code here, I don't want to refactor the whole module for a name change.

It wasn't my intention to suggest whole module refactor. I missed that get_process_info/2 wasn't added by you.